deploy/charts/harvester/dependency_charts/kubevirt/values.yaml

```diff
@@ -102,9 +102,13 @@ spec:
     # cpuRequest: "100m"
 
     ## Specify the machine type regex for validating the emulation of VirtualMachineInstance,
-    ## defaults to discover automatically, "pseries*" if arch is ppc64le, otherwise "q35*,pc-q35*".
+    ## defaults to discover automatically, "pseries*" if arch is ppc64le, otherwise "q35, pc-g35*, pc, pc-i440fx*".
     ##
-    # emulatedMachines: []
+    emulatedMachines:
+    - q35
+    - pc-q35*
+    - pc
+    - pc-i440fx* 
```

Comment:
Let's put customized values to `harvester/values.yaml` instead of the values file in the sub chart.
Add `pseries*` for ppc64le arch and s/q35/q35*/ to keep consistent with the original values(otherwise pseries will fail the validation).